content/en/events/kcc2021/faq.md

```diff
@@ -141,6 +141,33 @@ overlays. A few examples of what this would look like:
 <br>
 <img align="center" src="/events/kcc2021/name.png" width="40%">
 
+### Information on each channel in Discord
+
+- `announcements` : Announcements for any new events/updates will be done here
+- `code-of-conduct` : Make sure to accept the code of conduct by reacting with the Kubernetes emoji to the message!
+- `instructions-to-connect` : Follow the steps mentioned therre to enable all the existing channels
+- `faq` : Have a question or having a problem? Your question may be answered here :)
+- `help` : Need help? Have a question? Ask them here!
+- `feedback` : We would love to here your thoughts on what we can improve on and we can do a better job next time.
+- `general` : General chat for folks to chat
+- `awwwwww` : Show us pictures of your pets and other fluffy things that are cute. Humans allowed.
+- `food` : Show us food pictures and other tasty items around you.
+- `lfg` : Looking for a people to play a game with? Post your invite here!
+- `games` : Share links to play online games with each other.
+- `board-games` : Any board games enthusiast, this is your place to be!
+- `theater` : Let's watch movies and stuff and chat about it here.
+- `links-from-voice` : Use this channel to share links from various voice channels.
+- `main-events-chats` :  Text discussion related to events taking place in the main event channel
```

Comment:
```suggestion
- `main-events-chat` :  Text discussion related to events taking place in the main event channel
```
my mistake in initial suggestion had an s in there, will fix that quick in the GitHub UI